python/pyspark/pandas/utils.py

```diff
@@ -467,11 +467,18 @@ def is_testing() -> bool:
 
 def default_session() -> SparkSession:
     spark = SparkSession.getActiveSession()
-    if spark is not None:
-        return spark
+    if spark is None:
+        spark = SparkSession.builder.appName("pandas-on-Spark").getOrCreate()
+
+    if spark.conf.get("spark.sql.ansi.enabled"):
+        log_advice(
+            "The config 'spark.sql.ansi.enabled' is set to True."
```

Comment:
nit: I guess we need an space at the end of this string.

```
"... set to True. "
```

Otherwise, the two sentences are connected without a space like " ... set to True.This ..."

Reply:
Thanks!